ReportPortal.ATM/TestProject/Features/CreateAndEditDashboard.feature

```diff
@@ -0,0 +1,31 @@
+﻿Feature: Dashboard Creation and Edit feature
```

Comment:
* Scneario's name should follow test case's name recommendations, e.g.:
    * Verify the user can create a new dashboard.
    * Verify the user can edit a dashboard.